test/metabase/lib/metric_test.cljc

```diff
@@ -211,6 +231,13 @@
                     :aggregation-position 0}]
                   (map #(lib/display-info query-with-metric %)
                        metrics)))))))
+
+  (testing "Should return the available metrics as sorted"
+    (let [query   (lib/query metadata-provider-with-multiple-metrics (meta/table-metadata :venues))
+          metrics (lib.metric/available-metrics query)]
+      (is (=? (:id (first metrics)) second-metric-id))
+      (is (=? (:id (second metrics)) metric-id))))
```

Comment:
nit: I believe `(=? [{:id second-metric-id} {:id metric-id}] metrics)` could be used here.